pkg/volume/hostpath/host_path_test.go

```diff
@@ -196,7 +198,9 @@ func TestProvisioner(t *testing.T) {
 		t.Errorf("Expected reclaim policy %+v but got %+v", v1.PersistentVolumeReclaimDelete, pv.Spec.PersistentVolumeReclaimPolicy)
 	}
 
-	os.RemoveAll(pv.Spec.HostPath.Path)
+	pvDir := filepath.Dir(pv.Spec.HostPath.Path)
+	os.RemoveAll(pvDir)
```

Comment:
This can be simplified to `hostPathCreator.basePath`

Reply:
done, thanks